### DISCUSSION TITLE
How do I use .unityweb files?

### DISCUSSION THREAD

**Discussion by zhao-stanley** [jeffreylanters/react-unity-webgl#433] [Help Wanted!]
Hey! Are .unityweb files supported? I have a JSON file that points to these .unityweb files, so I was wondering if they would work with this package. Also, I get a warning on NextJS about the id:
![image](https://user-images.githubusercontent.com/101726102/186692215-5400095b-7926-4aa0-b115-43da52605418.png)

**Comment by zhao-stanley** [comment]
Couldn't find any up-to-date answers to the .unityweb question in the discussions. My config:
![image](https://user-images.githubusercontent.com/101726102/187251088-713545b3-b651-4ef5-9e6a-d9a91109dbc7.png)

**Comment by yv3nne** [comment]
It worked fine once I removed .unityweb from all of the files
